pkgs/build-support/vm/windows/default.nix

```diff
@@ -22,7 +22,7 @@ in {
   runInWindowsVM = drv: let
     newDrv = drv.override {
       stdenv = drv.stdenv.override {
-        shell = "/bin/sh";
+        shell = "${stdenv.shell}";
```

Comment:
Yes, this is deliberate, because this is within a VM and within a cygwin shell and there is no access to the store.